README.md

```diff
@@ -441,17 +444,20 @@ open-attestation token-registry issue --network <NETWORK> --address <TOKEN_REGIS
 ```
 >**Important:** In this command, you can use `mint` instead of `issue` and they will be strictly equivalent.
 
-See more options [here](#providing-the-wallet).
+For more ways to provide the wallet, see [here](#providing-the-wallet).
 
-The following command is a recommended example with the private key set in the `OA_PRIVATE_KEY` environment variable. 
+In the example below: 
+* You will use 0x6FFeD6E6591b808130a9b248fEA32101b5220eca as beneficiary and holder.
+* You will need to replace the value above with a wallet address you control, to be able to perform different actions on the transferable records later.
+* The `key.txt` file stores the private key to token registry.
 
 ```bash
-open-attestation token-registry mint --network sepolia --address 0x6133f580aE903b8e79845340375cCfd78a45FF35 --tokenId 0x10ee711d151bc2139473a57531f91d961b639affb876b350c31d031059cdcc2c --to 0xB26B4941941C51a4885E5B7D3A1B861E54405f90
+open-attestation token-registry issue -a 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --tokenId 0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea --beneficiary 0x6FFeD6E6591b808130a9b248fEA32101b5220eca --holder 0x6FFeD6E6591b808130a9b248fEA32101b5220eca -n sepolia -f key.txt
 ```
 The response looks like:
 
 ```
-✔  success   Token with hash 0x10ee711d151bc2139473a57531f91d961b639affb876b350c31d031059cdcc2c has been issued on 0x6133f580aE903b8e79845340375cCfd78a45FF35 with the initial recipient being 0xB26B4941941C51a4885E5B7D3A1B861E54405f90
+✔  success   Token with hash 0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea has been issued on 0x1E63411DC2fCd6Fab5EE938622f5f6A390F48272 with the initial recipient being 0x6FFeD6E6591b808130a9b248fEA32101b5220eca and initial holder 0x6FFeD6E6591b808130a9b248fEA32101b5220eca
```

Comment:
I think there is an error here as well as in the current `Issuing Transferable Record` page. The address supplied above should be the same as the one in the expected output.

e.g. in the above command we pass in 
```
... open-attestation token-registry issue -a 0x8431012Bc040942B59e3C5bf428221eab0b2f723 ...
```

the response should be
```
... has been issued on 0x8431012Bc040942B59e3C5bf428221eab0b2f723 ...
```
instead of `0x1E63411 ...`

Reply:
updated in both places